builders/ethereum/json-rpc/moonbeam-custom-api.md

```diff
@@ -7,7 +7,12 @@ description: Discover Moonbeam's specialized API endpoints, featuring custom JSO
 
 ## Introduction {: #introduction }
 
-Moonbeam nodes include support for two custom JSON-RPC endpoints: `moon_isBlockFinalized` and `moon_isTxFinalized`. These endpoints provide valuable functionality for checking the finality of on-chain events.
+Moonbeam nodes include support for custom JSON-RPC endpoints: 
```

Comment:
```suggestion
Moonbeam nodes include support for custom JSON-RPC endpoints: 

```

Reply:
<img width="979" alt="Screenshot 2024-08-07 at 9 21 56 AM" src="https://github.com/user-attachments/assets/2ef80b42-e82a-43fb-8171-26aad44eca4a">
Currently looks like this but the extra line should fix it.